src/main/java/com/esri/samples/na/find_service_areas_for_multiple_facilities/README.md

```diff
@@ -0,0 +1,38 @@
+# Find service areas for multiple facilities
+
+Find the service areas of several facilities from a feature service.
+
+![](FindServiceAreasForMultipleFacilities.png)
+
+## Use case
+
+A service area is a region which can be accessed from a facility as limited by one or more factors, such as travel time, distance, or cost. When analyzing the service area of multiple facilities, this workflow can be used to identify gaps in service area coverage, or significant overlaps, helping to optimize the distribution of facilities. For example, a city's health service may identify areas of a city that can be served effectively from particular hospitals, and with this optimize distribution of staff and resources.
+
+## How to use the sample
+
+Click 'find service areas' to calculate and display the service area of each facility on the map. The polygons displayed around each facility represents the service area that is within 3 minutes away from the hospital by car.
```

Comment:
this needs updated to explain there is also a polygon showing 5 minutes away by car.

Reply:
We should also put in a brief sentence to explain each facility's service area may overlap with another facility's service area.